snuba/datasets/configuration/generic_metrics_v2/entities/gauges.yaml

```diff
@@ -0,0 +1,161 @@
+version: v1
+kind: entity
+name: generic_metrics_gauges_v2
+
+schema:
+  [
+    { name: org_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: metric_mri, type: String },
+    { name: timestamp, type: DateTime },
+    { name: bucketed_time, type: DateTime },
+    { name: granularity, type: UInt, args: { size: 16 } },
+    { name: use_case_id, type: String },
+    {
+      name: tags,
+      type: Nested,
+      args:
+        {
+          subcolumns:
+            [
+              { name: key, type: String },
+              { name: value, type: String },
+            ],
+        },
+    },
+    {
+      name: min,
+      type: AggregateFunction,
+      args: { func: min, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: max,
+      type: AggregateFunction,
+      args: { func: max, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: sum,
+      type: AggregateFunction,
+      args: { func: sum, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: count,
+      type: AggregateFunction,
+      args: { func: count, arg_types: [{ type: UInt, args: { size: 64 } }] },
+    },
+    {
+      name: last,
+      type: AggregateFunction,
+      args: { func: count, arg_types: [{ type: Float, args: { size: 64 } }] },
```

Comment:
This was a typo I made on the original gauges entity. If I understand correctly, these args only affects the name of the aggregate function and name of types that's being passed around (not the actual Clickhouse aggregate function being used under the hood). So I think it should be

```
{
          func: argMax,
          arg_types: [{ type: Float, args: { size: 64 } }, { type: DateTime }],
}
```